docs/src/api.md

```diff
@@ -6,6 +6,7 @@ Documentation for AdvancedMH.jl
 ## Structs
 ```@docs
 MetropolisHastings
+Transition
```

Comment:
Do we ever want users to deal with `Transition`?

Reply:
Probably not.. I can pull that out